inc/admin/classes/class-wp-statuses-admin.php

```diff
@@ -88,7 +88,7 @@ private function hooks() {
 		if ( function_exists( 'register_block_type' ) )  {
 			add_action( 'init',                        array( $this, 'register_block_editor_script' ), 1001 );
 			add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_block_editor_asset' ), 10 );
-			add_filter( 'block_editor_preload_paths',  array( $this, 'preload_path' ), 10 );
+			add_filter( 'block_editor_rest_api_preload_paths',  array( $this, 'preload_path' ), 10 );
```

Comment:
```suggestion
			if ( function_exists( 'block_editor_rest_api_preload' ) ) {
				add_filter( 'block_editor_rest_api_preload_paths', array( $this, 'preload_path' ), 10 );
			} else {
				add_filter( 'block_editor_preload_paths', array( $this, 'preload_path' ), 10 );
			}
```

Could you add this check to be sure we stay back compatible with WordPress < 5.8 ?

Reply:
Makes perfect sense, added